hw_diag/views/diagnostics.py

```diff
@@ -53,18 +53,21 @@ def get_diagnostics():
         display_lte=display_lte
     )
 
+
 def is_gwmfr_running():
     return not os.path.isfile(ECC_SUCCESSFUL_TOUCH_FILEPATH)
 
+
 @DIAGNOSTICS.route('/initFile.txt')
 def get_initialisation_file():
     """
     This needs to be generated as quickly as possible,
     so we bypass the regular timer.
     """
-    
+
     if is_gwmfr_running():
-        logging.info("hm-gwmfr is still runnning. initFile.txt will not be returned")
+        logging.info("hm-gwmfr is still runnning. \
+            initFile.txt will not be returned.")
```

Comment:
Just tested this. It will add some undesirable spacing but we can fix later.

I'm not actually sure the correct syntax. Is it?

```
logging.info("A" +
   "B")?

Reply:
i think that's fine. Worse case scenario the logging gets a newline.